python/common/org/Python.java

```diff
@@ -390,6 +390,34 @@ public static void initializeModule(java.lang.Class cls, java.util.Map<java.lang
         return kwargs;
     }
 
+    /**
+     * Check that the number of targets given for unpacking matches the number of provided values
+     */
+    public static void checkUnpackValues(org.python.types.List values, int tcount, int scount) {
+        int values_count = (int) ((org.python.types.Int) values.__len__()).value;
+        int min_target_count = tcount - scount;
+
+        if (scount == 0 && min_target_count < values_count) {
+            throw new org.python.exceptions.ValueError("too many values to unpack (expected " + String.format("%d", min_target_count) + ")");
+        }
+        if (org.Python.VERSION < 0x03050000) {
+            if (min_target_count > values_count) {
+                if (values_count == 1) {
+                    throw new org.python.exceptions.ValueError("need more than 1 value to unpack");
+                } else {
+                    throw new org.python.exceptions.ValueError("need more than " + String.format("%d", values_count) + " values to unpack");
```

Comment:
Why are you going through `String.format()` step instead of just concatenating the integer value onto the string?

Reply:
Left over from a copy-and-paste.  I will clean it up.

Reply:
I cleaned this up, after which I recognized and implemented some additional improvements to this portion of the code.